packages/clay-core/src/typography/__tests__/index.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {Heading, Text} from '..';
+import {cleanup, render} from '@testing-library/react';
+import React from 'react';
+
```

Comment:
Just to add some test scenarios here for this component is to add the text for the default rendering without setting the properties.